src/swagger/specGenerator.ts

```diff
@@ -100,7 +100,11 @@ export class SpecGenerator {
     const pathMethod: Swagger.Operation = pathObject[method.method] = this.buildOperation(controllerName, method);
     pathMethod.description = method.description;
     pathMethod.summary = method.summary;
-    pathMethod.tags = method.tags;
+    
+    pathMethod.tags = [controllerName].concat(method.tags || []) 
```

Comment:
What does this do?

Reply:
This creates a default tag based on the controller name. I'm using it to group methods by controller name. I will remove it from the pull request as it is another modification that isn't related to operationId.